.gitignore

```diff
@@ -22,6 +22,9 @@
 /cover/
 /doc/
 
+# Temporary files for e.g. tests.
+/tmp/
```

Comment:
Do we have such tests in TypedStruct? I doubt so. If I wanted to update the `.gitignore` it would be more to delete unneeded lines. This was generated by `xgen`, my project generator, so if I add some non-project-specific line, it would be more in my project generator than here.

Reply:
> Do we have such tests in TypedStruct? I doubt so. If I wanted to update the `.gitignore` it would be more to delete unneeded lines. This was generated by `xgen`, my project generator, so if I add some non-project-specific line, it would be more in my project generator than here.

Didn't know about the `xgen`. The extra rule was copied from the `.gitignore` generated by latest Elixir version 1.11.